example/crd/bar_crd.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
How about s/bar_crd.yaml/bar.yaml

As this file was already under `crd` folder

Reply:
The crd suffix shows that it contains CRD definition, opposite to creation of CRD instances.